frontend/src/components/browse-sightings-page/MapPage.tsx

```diff
@@ -0,0 +1,43 @@
+import { Sighting } from "../../clients/apiClient";
+// import { SlippyMap, Marker, Label, InfoBox } from "react-slippy-map";
+import GoogleMapReact from "google-map-react";
+import { Marker } from "./marker";
```

Comment:
As below, capitalise component files (and you will almost certainly have trouble with this)
```suggestion
import { Marker } from "./Marker";
```